SpatialGDK/Documentation/content/glossary.md

```diff
@@ -1,33 +1,397 @@
+
+### SpatialOS SDK
+This is a set of low-level tools in several programming languages which you can use to integrate your game project with SpatialOS. It consists of the Worker SDK (or “Worker module”) and Platform SDK (or “Platform module”). 
+If you are using Unreal, you do not need to use the Worker SDK or Platform SDK, however, you can use the Worker SDK to extend or compliment the functionality of the GDK for Unreal. 
+
+> Related:
+>
+> Worker SDK: [Game development tools overview](https://docs.improbable.io/reference/latest/shared/dev-tools-intro)
+> [Platform SDK overciew](https://docs.improbable.io/reference/13.3/platform-sdk/introduction)
+
+### SpatialOS world
+
+>Not to be confused with the Unreal [game world](#game-world).
+
+Also known as "the world".
+The world is a central concept in SpatialOS. It’s the canonical source of truth about your game. All the world's data is stored within [entities](#entity) - specifically, within their [components](#component).
+SpatialOS manages the world, keeping track of all the entities and what state they’re in.
+Changes to the world are made by [workers](#worker). Each worker has a view onto the world (the part of the world that they're [interested](#interest) in), and SpatialOS sends them updates when anything changes in that view.
+It's important to recognise this fundamental separation between the SpatialOS world and the view of that world that a worker [checks out](#check-out). Workers send updates to SpatialOS when they want to change the world: they don't control the canonical state of the world, they must use SpatialOS APIs to change it.
+
+### Snapshot
+A snapshot is a representation of the state of a [SpatialOS world](#spatialos-world) at a given point in time. It stores each [persistent](#persistence) [SpatialOS entity](#spatialos-entity) and the values of their [SpatialOS components](#spatialos-component)' [properties (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#property).
+You'll use a snapshot as the starting point (using an an “initial snapshot”) for your [SpatialOS world](#spatialos-world) when you [deploy your game](#deployment).
+
+> Related:
+> 
+> * [How to generate a snapshot]({{urlRoot}}/content/generating-a-snapshot) (GDK documentation)
+> * [Snapshots (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/operate/snapshots)
+
+### Streaming queries
+Streaming queries allow workers to get information about the [world](#spatialos-world) outside the region they’re [interested in](#interest), so that they know about entities that they don’t have checked out (for example, entities that are far away, or that don’t have a physical position).
+
+> Streaming queries are useful if you need to get information about an entity periodically - for example, so that a player can see and interact with it.
+> If you just need information about an entity at one particular time, use [queries](#queries) instead.
```

Comment:
`use [queries](#queries) instead.` 
This should be `use [entity queries](#queries) instead.`